### ISSUE TITLE
qw state

### ISSUE THREAD

**Issue by lurebgi** [jtriley/StarCluster#550]
I was using c1.xlarge node instance, and I added 6 nodes. However, my job was still at 'qw' state after 12 hours waiting. Is thit normal?

btw, the previous issue was 'the requested parallel environment "smp" does not exist', so I modified the configure according to 
http://stackoverflow.com/questions/21123513/sge-uge-etc-standardized-way-to-submit-openmp-jobs-to-multiple-cores

Can this be problematic?

**Comment by cariaso** [comment]
yes, your configure seems overly complicated. The name you're submitting to should be 'orte' not 'smp'. Make just that change and everything should flow easily.

**Comment by lurebgi** [comment]
Thanks, 'orte' works well.
